src/bp/lang-server/service/index.ts

```diff
@@ -41,10 +62,34 @@ export default class LanguageService {
     this._models = this._getModels()
     const languages = Object.keys(this._models)
 
+    if (languages.length > 0) {
+      const ramInfos = checkEnoughRam(languages.length, this.dim)
+
+      if (ramInfos.prediction + ramInfos.prediction * MARGIN_PRED > ramInfos.free) {
+        this.logger.warn(
+          'Warning, this program can crash silently if it lacks free memory space. Make you sure you have enough RAM available.'
+        )
+        this.logger.warn(
+          `Lang server will take ${ramInfos.prediction}Gb and you have approximately ${ramInfos.free}Gb left on ${ramInfos.total}`
+        )
+      }
```

Comment:
this warning is useless in my opinion. This should not be a warning, it's really informative, change .warn to .info please